### PR TITLE
Adding the PublicLock interface

### PR DESCRIPTION
**Pull request by nfurfaro**
# Description
This is the first step towards fixing issue 5008.
The next step will be to add the interface to the module.

# Issues

Fixes #
Refs #5008

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [x] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [x] My code follows the style guidelines of this project, including naming conventions
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [x] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by NickCuso** [comment]
Could you use this interface from PublicLock.sol?  That will help to confirm it's correct.
Additionally, maybe longer term, it would be nice to test this to confirm it's complete.

**Comment by nfurfaro** [comment]
oops, yes, I did test with the interface imported into `PublicLock`, but forgot to commit it!

File: smart-contracts/contracts/PublicLock.sol [hunk 1]
```diff
@@ -2,6 +2,7 @@ pragma solidity 0.5.12;
 
 import './interfaces/IERC721.sol';
 import './interfaces/IERC721Enumerable.sol';
+import './interfaces/IPublicLock.sol';
```
Comment (NickCuso) [inline smart-contracts/contracts/PublicLock.sol]: Need to add this to the `is` list as well ;)

Comment (nfurfaro) [reply smart-contracts/contracts/PublicLock.sol]: Doh! Sorry, that's what happens when I'm trying to work while cooking dinner and helping kids with homework. Will fix and resubmit.

**Pull request merged**